authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/impl/idevent/CredentialStoreServiceImpl.java

```diff
@@ -324,7 +322,7 @@ private IdentityEntity doProcessCredentialStoreEvent(CredentialEventStore creden
 		
 		String eventObjectStr = credentialEventStore.getEventObject();
 		try {
-			mosipLogger.info(IdAuthCommonConstants.SESSION_ID, this.getClass().getName(), "processCredentialStoreEvent",
+			mosipLogger.debug(IdAuthCommonConstants.SESSION_ID, this.getClass().getName(), "processCredentialStoreEvent",

```

Comment:
Let's remove the debug statement. Event Store data might contain sensitive information.